### ISSUE TITLE
Can't Add new category on Select Category screen

### ISSUE THREAD

**Issue by workerbee22** [suesitran/my_wallet#35]
When I'm selecting a Category for an Expense, I look at the one that are there, but expect to be able to add a new one. When starting with the app, I gradually add new categories as I think of them. That is, attempt to select category, note that none of my existing categories apply and want to add a new one.

**Comment by suesitran** [comment]
agree, this will make the flow easier for user to manage his category listing.
